src/ContentHubClientTrait.php

```diff
@@ -235,4 +250,39 @@ public function ping() {
     return $client->get('ping');
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function send(RequestInterface $request, array $options = []): ResponseInterface {
+    return $this->httpClient->send($request, $options);
```

Comment:
Wouldn't this throw an error calling send on null? and same for other 4 methods since httpClient might not be set. Here it becomes responsibility of class using this trait to initialise httpClient.
@saxumVermes what do you think?

Reply:
That's right. The trait proposes a contract which must be satisfied by the implementing classes. ContentHubCientTrait is very specific in this case. The responsibility is delegated to the class in hand.
The most appropriate way to handle this problem is to provide the client as an argument, however that would break backward compatibility.